<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="7" hostname="DESKTOP-3JC72G1" name="completeTest.NopCommerceOrderCompletion" tests="10" failures="0" timestamp="2023-06-13T19:33:13 IST" time="2.872" errors="3">
  <testcase name="setUp" time="0.591" classname="completeTest.NopCommerceOrderCompletion">
    <skipped/>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase name="screenshot" time="0.331" classname="completeTest.NopCommerceOrderCompletion">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.resources.BaseClass.screenShotPage(BaseClass.java:222)
at completeTest.NopCommerceOrderCompletion.screenshot(NopCommerceOrderCompletion.java:65)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
    </error>
  </testcase> <!-- screenshot -->
  <system-out/>
  <testcase name="closeSession" time="0.523" classname="completeTest.NopCommerceOrderCompletion">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at completeTest.NopCommerceSetup.tearDown(NopCommerceSetup.java:25)
at completeTest.NopCommerceOrderCompletion.closeSession(NopCommerceOrderCompletion.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- closeSession -->
  <system-out/>
  <testcase name="shopCartPage" time="0.508" classname="completeTest.NopCommerceOrderCompletion">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.resources.BaseClass.reload(BaseClass.java:137)
at completeTest.NopCommerceOrderCompletion.shopCartPage(NopCommerceOrderCompletion.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
    </error>
  </testcase> <!-- shopCartPage -->
  <system-out/>
  <testcase name="checkOut" time="0.475" classname="completeTest.NopCommerceOrderCompletion">
    <skipped/>
  </testcase> <!-- checkOut -->
  <system-out/>
  <testcase name="billingAddressPage" time="0.444" classname="completeTest.NopCommerceOrderCompletion">
    <skipped/>
  </testcase> <!-- billingAddressPage -->
  <system-out/>
  <testcase name="shipMethodPage" time="0.000" classname="completeTest.NopCommerceOrderCompletion">
    <skipped/>
  </testcase> <!-- shipMethodPage -->
  <system-out/>
  <testcase name="paymentMethodPage" time="0.000" classname="completeTest.NopCommerceOrderCompletion">
    <skipped/>
  </testcase> <!-- paymentMethodPage -->
  <system-out/>
  <testcase name="paymentInfoPage" time="0.000" classname="completeTest.NopCommerceOrderCompletion">
    <skipped/>
  </testcase> <!-- paymentInfoPage -->
  <system-out/>
  <testcase name="confirmOrderPage" time="0.000" classname="completeTest.NopCommerceOrderCompletion">
    <skipped/>
  </testcase> <!-- confirmOrderPage -->
  <system-out/>
</testsuite> <!-- completeTest.NopCommerceOrderCompletion -->
